Scripts DICOM/writeTROTSToDICOM.py

```diff
@@ -215,6 +225,317 @@
                         cont.ContourImageSequence.append(ci)
                         rc.ContourSequence.append(cont)
             rds.ROIContourSequence.append(rc)
-            
 
         rds.save_as("./DICOMs/"+patientFolder+'/rtstruct.dcm', write_like_original = False)
+        
+        if((folder == 'Protons') and ('beamlistfolder' in locals()) and ('machinedata' in locals())):
+            # write rtplan
+            print('Working on rtplan...')
+            beamSigmaEnergy = machinedata["BeamInfo"]["EnergyRangeList"]
+            beamSigmas = machinedata["BeamInfo"]["Sigma"]
+            SAD = [machinedata["BeamInfo"]["SAD"][0],machinedata["BeamInfo"]["SAD"][1]]
+            meta = pydicom.Dataset()
+            SOPInstanceUID = pydicom.uid.generate_uid()
+            meta.MediaStorageSOPClassUID = pydicom.uid.RTIonPlanStorage
+
+            meta.MediaStorageSOPInstanceUID = SOPInstanceUID
+            meta.TransferSyntaxUID = pydicom.uid.ExplicitVRLittleEndian  
+            
+            rtds = Dataset()
+            rtds.file_meta = meta
+            rtds.SOPClassUID = pydicom.uid.RTPlanStorage

```

Comment:
```suggestion
            rtds.SOPClassUID = pydicom.uid.RTIonPlanStorage
```
you lost this change while rebasing maybe?